test/AppSpec.hs

```diff
@@ -22,23 +21,22 @@ spec = do
     withClient mkApp $ do
       it "lists an example item" $ \ env -> do
         try env getItems `shouldReturn` [Item 0 "example item"]
-
       it "allows to show items by id" $ \ env -> do
         try env (getItem 0) `shouldReturn` Item 0 "example item"
-
       it "throws a 404 for missing items" $ \ env -> do
-        try env (getItem 42) `shouldThrow` (\ e -> responseStatus e == notFound404)
+        try env (getItem 42) `shouldThrow` anyException
```

Comment:
Yeah, `anyException` doesn't make sure that this actually throws a 404, which is what the test is supposed to test. What was the problem with the previous code?